flink-table/flink-sql-client/src/test/resources/sql/function.q

```diff
@@ -406,7 +406,7 @@ describe function extended temp_upperudf;
 |              requirements |                                         $VAR_UDF_JAR_PATH_SPACE [] |
 |          is deterministic |                                       $VAR_UDF_JAR_PATH_SPACE true |
 | supports constant folding |                                       $VAR_UDF_JAR_PATH_SPACE true |
-|                 signature |                $VAR_UDF_JAR_PATH_SPACE c1.db.temp_upperudf(STRING) |
+|                 signature |        $VAR_UDF_JAR_PATH_SPACE c1.db.temp_upperudf(arg0 => STRING) |
```

Comment:
Does this mean that the function takes a function producing a string?  

This is a surprising change to me.

Reply:
No, this is the SQL syntax for named arguments.